podvm/qcow2/centos/variables.pkr.hcl

```diff
@@ -21,12 +21,12 @@ variable "disk_size" {
 
 variable "cloud_image_checksum" {
   type    = string
-  default = "b80ca9ccad8715aa1aeee582906096d3190b91cebe05fcb3ac7cb197fd68a7fe"
+  default = "d12bb6934dd207e242d6aa13f6a4ca4969449c14c3bbdd88a5ce5f5203597a40"
```

Comment:
Not for this PR, wondering if it is possible to read the checksum from `versions.yaml`, is it?

Reply:
I thought we may be able to just drop the defaults instead, need to verify it doesn't break anything, i thought to do it later

Reply:
cool, it would be nice to remove that if it is a simple duplication of code.